README.md

```diff
@@ -293,6 +297,48 @@ spec:
   ...
 ```
 
+#### Google Container Builder plugin
+
+You need to create a Google Cloud service account JSON with "Project Editor" role ([?](https://cloudplatform.googleblog.com/2018/03/automatic-serverless-deployments-with-Cloud-Source-Repositories-and-Container-Builder.html)) in https://console.cloud.google.com/iam-admin/serviceaccounts , and create a Kubernetes secret as follows:
```

Comment:
You should be able to get by with "Cloud Container Builder Editor": https://cloud.google.com/container-builder/docs/securing-builds/configure-access-control

Reply:
Unfortunately it did not work
```
Uploading tarball of [/cbi-gitcontext/context] to [gs://....tgz]
Created [https://cloudbuild.googleapis.com/v1/projects/...].
Logs are available at [https://console.cloud.google.com/gcr/builds/....
ERROR: (gcloud.container.builds.submit) HTTPError 403: <?xml version='1.0' encoding='UTF-8'?><Error><Code>AccessDenied</Code><Message>Access denied.</Message><Details>...@....iam.gserviceaccount.com does not have storage.objects.get access to ....cloudbuild-logs.googleusercontent.com/log-....txt.</Details></Error>
```

I also set `Cloud Container Builder Viewer` and `Storage Admin` to the role, but still getting the same error.

Reply:
Oh, sorry, in that case the user also needs `Project Viewer`. :-/

Reply:
Thanks, `Cloud Container Builder Editor`+`Project Viewer`+`Storage Admin` works for me.

Without `Storage Admin`, I got an error: `ERROR: (gcloud.container.builds.submit) HTTPError 403: ****@****.iam.gserviceaccount.com does not have storage.buckets.create access to project ****`. 

@ImJasonH If you are working on the GCR team, could you add `Cloud Container Builder Editor`+`Project Viewer`+`Storage Admin` to the official documentation?
cc @mattmoor